public/views/transaction/tx.html

```diff
@@ -1,68 +1,21 @@
-      <span class="lead glyphicon glyphicon-arrow-down text-muted" data-ng-class="{
-        'text-danger': tx.type == 0 && $parent.address.address == tx.senderId && tx.senderId != tx.recipientId,
-        'text-success': tx.type == 0 && $parent.address.address && tx.senderId != $parent.address.address,
-        'text-muted': tx.type != 0
-      }"></span>
-    </div>
-  </div>
-  <div class="col-md-6">
-    <div class="row">
-      <div>
-        <div class="panel panel-default">
-          <div class="panel-body">
-            <div class="pull-right">{{tx.amount | currency:currency}} {{currency.symbol}}</div>
-            <div class="ellipsis" data-ng-show="tx.type == 0">
-              <a href="/address/{{tx.recipientId}}">{{tx | txRecipient}}</a>
-            </div>
-            <div class="ellipsis" data-ng-show="tx.type >= 1">{{tx | txType}}</div>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
-</div>
-<div class="line-top row" data-ng-hide="!tx">
-  <div class="col-xs-12 col-sm-4 col-md-4">
-    <span class="txvalues txvalues-default">Fee: {{tx.fee | currency:currency}} {{currency.symbol}}</span>
-  </div>
-  <div class="col-xs-12 col-sm-8 col-md-8 text-right">
-    <span data-ng-if="block.confirmations > 0 || tx.confirmations > 0" class="txvalues txvalues-success">{{block.confirmations || tx.confirmations}} Confirmations</span>
-    <span data-ng-if="!block.confirmations && !tx.confirmations" class="txvalues txvalues-danger">Unconfirmed Transaction!</span>
-    <span class="txvalues txvalues-primary">{{tx.amount | currency:currency}} {{currency.symbol}}</span>
-  </div>
-</div>
+	<td data-title="Transaction ID">
+		<a class="txid" href="/tx/{{tx.id}}">{{tx.id}}</a>
+		<span class="btn-copy" clip-copy="tx.id"></span>
+	</td>
+	<td data-title="Sender"><a href="/address/{{tx.senderId}}">{{tx | txSender}}</a></td>
+	<td data-title="Recipient" data-ng-show="tx.type == 0"><a href="/address/{{tx.recipientId}}">{{tx | txRecipient}}</a></td>
+	<td data-title="Recipient" data-ng-show="tx.type >= 1">{{tx | txType}}</td>	
+	<td data-title="Amount">
+		<div class="txvalues" style="width:100%" data-ng-class="{
+      'txvalues-danger': tx.type == 0 && $parent.address.address == tx.senderId && tx.senderId != tx.recipientId,
+      'txvalues-success': tx.type == 0 && $parent.address.address && tx.senderId != $parent.address.address,
+      'txvalues-default': tx.type != 0
+    }">{{tx.amount | currency:currency}}{{currency.symbol}}
+    	</div>
+    </td>
+    <td data-title="Fee">{{tx.fee | currency:currency}}{{currency.symbol}}</td>
+	<td data-title="Confirmations">
+		<span data-ng-if="!block.confirmations && !tx.confirmations" class="text-danger">Unconfirmed Transaction!</span>
+		<span data-ng-if="block.confirmations > 0 || tx.confirmations > 0" class="text-success">{{block.confirmations || tx.confirmations}}</span>
+	</td>
+	<td data-title="Day">{{tx.timestamp | timeAgo }}</td>
```

Comment:
Also we should drop `timeAgo` filter here and display normal date - both for mobile and desktop.